### DISCUSSION TITLE
CI problem

### DISCUSSION THREAD

**Discussion by jkotlinski** [jkotlinski/durexforth#529] [Q&A] [answered]
CI builds started failing, because the URL to headless VICE snapshot build is now outdated:

https://github.com/VICE-Team/svn-mirror/releases/download/r42928/headlessvice_3.7.1+r42928.deb

Not sure how to solve this in the best way, any ideas?

Obviously I could just add the binary to the Git repository, but that feels awkward -- binaries are not supposed to be added to source code repositories, right?

**Comment by Whammo** [comment]
Wasn't the binary in their Git repository?

**Comment by jkotlinski** [comment]
Only in temporary snapshot builds. I hope there will be a stable build in
future.

fre 10 mars 2023 kl. 19:51 skrev Whammo ***@***.***>:

**Reply by ekipan** [reply]
If they don't offer an official build it looks like your options are 1) bump the release whenever it falls off or 2) figure out how to build it yourself. But you probably already knew that.

Is it a problem to use, say, the SDL release instead of headless? I don't know anything about Git's CI.

**Comment by jkotlinski** [comment, marked as answer]
Resolved by forking svn-mirror, making a copy of the latest release there, and linking to that.
